package.json

```diff
@@ -6,10 +6,50 @@
     "node": "10.16.0"
   },
   "dependencies": {
-    "@babel/core": "7.5.5",
     "@material-ui/core": "^4.2.1",
     "@material-ui/icons": "^4.2.1",
-    "@svgr/webpack": "4.3.2",
+    "dotenv": "6.2.0",
+    "dotenv-expand": "4.2.0",
+    "notistack": "^0.4.3",
+    "react": "^16.8.0",
+    "react-app-polyfill": "^1.0.2",
+    "react-dom": "^16.8.0",
+    "react-redux": "^7.1.3",
+    "react-router-dom": "^5.1.2",
+    "react-scripts": "^3.0.1",
+    "redux": "^4.0.1",
+    "redux-mock-store": "^1.5.3",
+    "redux-thunk": "^2.3.0"
+  },
+  "scripts": {
+    "start": "node scripts/start.js",
+    "build": "node scripts/build.js",
+    "test": "jest --coverage --verbose",
+    "test-watch": "node scripts/test.js",
+    "lint-fix": "npm run lint -- --fix",
+    "lint": "eslint src --max-warnings 0"
+  },
+  "browserslist": [
+    ">0.2%",
+    "not dead",
+    "not ie <= 11",
+    "not op_mini all"
+  ],
+  "devDependencies": {
```

Comment:
This is a mess, really thinking that we maybe shouldn't have ejected CRA... 🙃

Reply:
The truth is devDependencies and normal dependencies are not separated in a React project, all of them are used in dev, and none in production, so I am not really concerned